src/ptf/testutils.py

```diff
@@ -2356,7 +2356,7 @@ def verify_packet_any_port(test, pkt, ports=[], device_number=0):
     Returns the index of the port on which the packet is received and the packet.
     """
     logging.debug("Checking for pkt on device %d, port %r", device_number, ports)
-    result = dp_poll(test, device_number=device_number, exp_pkt=pkt, timeout=1)
+    result = dp_poll(test, device_number=device_number, exp_pkt=pkt, timeout=1, port_number= ports)
```

Comment:
Could you elaborate a little bit on why the current fix is wrong? IMO To prevent having this warning when calling verify_packet_any_port, we should pass the right value to the port_number argument when it calls dp_poll.